docs/UserGuide.md

```diff
@@ -622,31 +666,23 @@ Examples:
 
 #### Viewing the overview of a project: `overview`
```

Comment:
Thinking of moving these 2 sections into projects

Reply:
Good idea. I'll shift it around.